### ISSUE TITLE
how long does it takes to train a 10000 images dataset.

### ISSUE THREAD

**Issue by alvinxiii** [thtrieu/darkflow#772]
First, I trained 3-5 images to overfit the model as mentioned by the creator. I used tiny yolo weight and cfg to train this dataset. Once it reached 3000 steps, I test the trained model and it successfully forms the bounding box on the object.

For the second step, once I confirm the model is working fine. Then, I used the trained model to train on my 10000 images dataset. I use the following parameters:
[python flow --train --model cfg/tiny-yolo-voc-1c.cfg --load -3000 --annotation train/annotation --dataset train/images --epoch 10 --gpu 1.0](url)
I train the dataset using K80 gpu card. When I resume the training at step 3000, the loss starts at 3.00 and it keeps remain at average 2 - 1.8 constantly around step 5000.
My question is, is my training is overfitting? Is it the right way to train such huge dataset. Thanks.

**Comment by youyuge34** [comment]
I strongly recommend u to use the full yolo model instead of tiny ones due to u have K80. The full model has a much better performance and will not overfit easily. 
I believe 40k steps are suitable and remember that the loss remains dont mean the model is converge, u should keep training.
Most important of all , use the command `keep -200` (or more) to save more `ckpt` files so u can just test different steps model. And stop the training for example at steps 10000, then load the last ckpt and use a smaller learning rate like 1e-6  to train continually is also significant.

**Comment by alvinxiii** [comment]
I found the full yolo weights here https://drive.google.com/drive/folders/0B1tW_VtY7onidEwyQ2FtQVplWEU
But where is the cfg file?

**Comment by Malouke** [comment]
youyuge34  ,
hi   i am wondering why you can give the complet ansewer ?
you tallk about tiny vs full but that still no prouve about overfitting .
my adavise is to incearse number of epochs to maximum possible .
what is your graphic cards and memory size if you have more than >10GO try to increase the numbers of batch genarly the best (32,64,256...) that increase accuracy  
try different trainer like adam.
it 's not about fully or tiny it just about the quality of input tray augmentation data too .
